### PR TITLE
Create Brand page

### LINKED ISSUE #677: Branding/Logos page
Hi everyone,

I've been looking for some logo usage guidelines and easily accessible logos on your website, but wasn't able to find them. 
Could you add a page with at least both logos (e only, and elementaryOS) in svg and some hints about the cases when they may be used (by press, support companies, or other software projects, etc.)?

(The reason I was looking for a logo was that our open source software deployment tool m23 (remote installation and client management via network to physical and virtual clients) now supports remote installation and updates/backups/package installation/etc. for elementaryOS, http://m23.sf.net).

I had asked the same question over at lp (https://bugs.launchpad.net/elementaryos/+bug/1457095), and got permission ;), but for the others who will search for this same thing, it may be useful to provide a page (saves you some answers ;) ).

### LINKED ISSUE #1036: Migrate docs copy styles to be more global
Any text-heavy pages (i.e. /privacy-policy) could benefit from the copy styles in the docs CSS. I propose we move those out of docs.css and into main.css.

### PR DESCRIPTION
**Pull request by cassidyjames**
Fixes #677. Also fixes #1036.

### CODE REVIEW

**Comment by cassidyjames** [comment]
Ready for someone to tear into it.

File: brand.php [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    <h2>Brand Marks</h2>
+    <p>elementary owns two marks: the "elementary" logotype and the "e" logomark. Both are considered trademarks and represent elementary—the company—and its products.</p>
+    <p>Both should be used with the following in mind:</p>
+    <ul>
+        <li>Do not stretch, skew, rotate, flip, or otherwise alter the marks.</li>
+        <li>Do not use the marks on an overly-busy background; solid colors work best.</li>
+        <li>The marks should always be monochromatic; typically white if on a dark background, or black if on a light background.</li>
+    </ul>
+
+    <h3>Logotype</h3>
```
Comment (cassidyjames) [inline brand.php]: Should add the logotype SVG here, eh?

File: brand.php [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    <p>The logotype is to be used when space allows to refer to elementary the company. It can be used before a product name to refer to a specific product of elementary.</p>
+    <p>The logotype should always be used under the following guidelines:</p>
+    <ul>
+        <li>Do not attempt to recreate the logotype. It is a meticulously-designed brand mark, not simply "elementary" written in a specific font.</li>
+        <li>Do not use the logotype at small sizes; if it is not clear, use the logomark instead.</li>
+    </ul>
+
+    <h3>Logomark</h3>
```
Comment (cassidyjames) [inline brand.php]: ...and a logomark SVG

File: brand.php [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    include '_templates/sitewide.php';
+    $page['title'] = 'Brand &sdot; elementary';
+    $page['scripts'] = '<link rel="stylesheet" type="text/css" media="all" href="styles/brand.css">';
+    include $template['header'];
+    include $template['alert'];
+?>
+
+<div class="row docs">
+    <h1>Brand</h1>
+    <p>The elementary brand is unique: technically it belongs to elementary LLC., the company that guides and supports development of elementary products. However, we have a great community and don’t want to be too overbearing with legal requirements and such. As such, we have written up some guidelines to make it easier to understand when and how the elementary brand should be used.</p>
```
Comment (danrabbit) [inline brand.php]: "and such. As such"

File: brand.php [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    <ul>
+        <li>You are encouraged to say that your app or service is designed for elementary OS, but please don't use the elementary name or marks as part of the name of your company, application, product, or service, or in any logo you create.</li>
+        <li>Only use the elementary name or marks to refer to elementary LLC. or its products (i.e. elementary OS).</li>
+    </ul>
+
+    <h2>Merchandise</h2>
+    <p>We do not typically allow our branding (including our name or brand marks) to be used on third-party merchandise.</p>
+
+    <h2>More Info</h2>
+    <p>For further info regarding the use of the elementary name, branding, and trademarks, please email brand@elementary.io.</p>
```
Comment (btkostner) [inline brand.php]: Include an actual link for the email? Could lead to more spam, but it's much more convenient for the reader.

File: brand.php (hunk 3 above)
Comment (cassidyjames) [inline brand.php]: Agree that's awkward. Suggestions? :D

File: brand.php (hunk 4 above)
Comment (cassidyjames) [inline brand.php]: Will do.

**Comment by danrabbit** [comment]
I think my only remaining concern is that you have `font-family: mono;` set instead of specifying roboto mono ;)

**Comment by cassidyjames** [comment]
Aha! Will fix.

**Comment by cassidyjames** [comment]
All set.

**Pull request merged**